tests/integration/test_dbapi_integration.py

```diff
@@ -1234,3 +1259,12 @@ def test_describe_table_query(run_trino):
             aliased=False,
         )
     ]
+
+
+def assert_cursor_description(cur, trino_type, size=None, precision=None, scale=None):
+    assert cur.description[0][1] == trino_type
+    assert cur.description[0][2] is None
```

Comment:
According to db-api the first two params are mandatory. Why do we assert here that type_code is None?